c/busybox-1.22.0/cal_false-unreach-call_true-no-overflow_true-valid-memsafety.c

```diff
@@ -1832,9 +1832,41 @@ ssize_t write(int fildes, const void *buf, size_t nbyte)
   return ret;
 }
 
+int getopt_long(int argc, char * const argv[], const char *optstring,
+                const struct option *longopts, int *longindex)
+{
+  (void)*longopts;
+  (void)longindex;
+
+  int result = -1;
+
+  if(optind >= argc || argv[optind][0] != '-')
+    return -1;
+
+  size_t opt_index = __VERIFIER_nondet_ulong();
```

Comment:
Maybe using `__VERIFIER_nondet_size_t` would be clearer. However, this should be also OK, since these benchmarks are 64-bit.